applications/GeoMechanicsApplication/custom_elements/transient_Pw_line_element.h

```diff
@@ -63,34 +71,72 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) TransientPwLineElement : public Elem
         rResult = Geo::DofUtilities::ExtractEquationIdsFrom(GetDofs());
     }
 
+    void Initialize(const ProcessInfo& rCurrentProcessInfo) override
```

Comment:
Since this function parameter is not being used, I would suggest to "unname" it:
```suggestion
    void Initialize(const ProcessInfo&) override
```

Reply:
Done